### DISCUSSION TITLE
Entrega de practica JS

### DISCUSSION THREAD

**Discussion by Gracesl** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#566] [Q&A]
@romarpla me apena hacer la pregunta, he enviado por instagram el mensaje de la posibilidad de aun entregar las ultimas practicas de ftront end, porfavor que me dieran oportunidad.

**Comment by doguedogue** [comment]
No es para mi la pregunta, pero te comento: 

Creo que es de lo que trató la platica de ayer de **no estresarse**, parece que mañana nos darán una mecánica para poner todo lo que hemos hecho (y hasta donde hayamos hecho)

En mi opinión personal, que lo hayas acabado eso ya te sirve a ti mucho.

Animo! 🚀

**Comment by Gracesl** [comment]
Gracias amigo Rafael relamente eso ea cierto al realizarlo ocurre la magia
de aprender.
